plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSplitSource.java

```diff
@@ -356,4 +369,154 @@ public void testNullStatisticsMaps()
                 ImmutableMap.of(),
                 ImmutableMap.of()));
     }
+
+    private IcebergSplitSource getCompletableSplitSource(IcebergTableHandle tableHandle, TableScan tableScan)
```

Comment:
Move helper methods down to right below where they get used

Reply:
Got it.